src/components/OptionsSelector.js

```diff
@@ -168,6 +171,7 @@ class OptionsSelector extends Component {
     }
 
     render() {
+        const {translations: {translate}} = this.props;
```

Comment:
For the class components, we want to avoid destructuring a single part of the props, so please just use `this.props.translate` instead. There are several instances of this throughout the diff. I know it's a hassle sorry 😅

Reply:
If the props are already destructured though, you're fine.

Reply:
Ok, I second-guessed myself here, and I could have sworn our style guide says that we don't want props and state destructured at the same time, but I'm not seeing that now. I think just to play this safe, you should use `this.props.translate` unless the props are already being destrucutred. Again, sorry for the inconvenience.